packages/lesswrong/server/ckEditor/ckEditorWebhook.ts

```diff
@@ -99,20 +153,52 @@ async function handleCkEditorWebhook(message: any) {
     case "comment.removed":
     case "commentthread.removed":
     case "commentthread.restored":
+    case "collaboration.user.connected": {
+      const userConnectedPayload = payload as CkEditorUserConnectionChange;
+      const userId = userConnectedPayload?.user?.id;
+      const documentId = userConnectedPayload?.document?.id;
+
+      const result = await createMutator({
+        collection: CkEditorUserSessions,
+        document: {
+          userId,
+          documentId
+        },
+        validate: false,
+      });
+
+      break
+    }
     case "document.user.connected":
-      break;
+    case "collaboration.user.disconnected": {
+      const userDisconnectedPayload = payload as CkEditorUserConnectionChange;
+      const userId = userDisconnectedPayload?.user?.id;
+      const documentId = userDisconnectedPayload?.document?.id;
```

Comment:
Neither of these should be missing, but if they are, it's very possible that we'd get totally wrong results from the subsequent `findOne` (since then it wouldn't be filtering based on both, but on one or neither, and would just return the first matching result).